phpunit.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you rename this file to `phpunit.xml.dist` and put `phpunit.xml` in `.gitignore` please?